### PR TITLE
Pr256/comments

### PR DESCRIPTION
**Pull request by alfetopito**
Addressing comments from https://github.com/gnosis/gp-ui/pull/256

### CODE REVIEW

File: src/api/deposit/batchExchangeAddresses.ts [hunk 1]
```diff
@@ -3,9 +3,9 @@ import { Network } from 'types'
 import { getGpV1ContractAddress } from 'utils/contract'
 
 const addressesByNetwork = {
-  [Network.Rinkeby]: getGpV1ContractAddress(Network.Rinkeby) as string,
-  [Network.Mainnet]: getGpV1ContractAddress(Network.Mainnet) as string,
-  [Network.xDAI]: getGpV1ContractAddress(Network.xDAI) as string,
+  [Network.Rinkeby]: getGpV1ContractAddress(Network.Rinkeby) || '',
```
Comment (W3stside) [inline src/api/deposit/batchExchangeAddresses.ts:6]: should the function just return empty string? anyways i think you know what you're doing here more than me so it's probably fine

File: src/components/layout/GenericLayout/Footer/index.tsx [hunk 2]
```diff
@@ -114,39 +114,39 @@ export const Footer: React.FC<FooterType> = (props) => {
...
             networkId={networkId}
-            label={'Allowance manager contract'}
+            label="Allowance manager contract"
           />
-        ) : null}
+        )}
       </ContractsWrapper>
       <VersionsWrapper>
-        {url.web && VERSION ? (
+        {url.web && VERSION && (
```
Comment (W3stside) [inline src/components/layout/GenericLayout/Footer/index.tsx:135]: noiiice

File: src/utils/contract.ts [hunk 3]
```diff
@@ -13,10 +13,10 @@ export function getContractAddressFromNetworkInfo(
   network: OptionalNetwork,
   networkInfo: ContractNetworkInfo,
 ): string | null {
-  if (!network) {
+  if (!network || networkInfo[network]?.address) {
```
Comment (W3stside) [inline src/utils/contract.ts:16]: mm i think this should just be: 
```suggestion
  if (!network || !networkInfo[network].address) {
```
or am i wrong? :thinking: now im confused with the coalescing

**Review by W3stside — APPROVED** [review]
just concerned about the utils/contracts line 16 and my noodle brain

File: src/utils/contract.ts (hunk 3 above)
Comment (alfetopito) [reply src/utils/contract.ts:16]: You are right, I forgot to negate the second part.
But maybe the coalescing should stay. Will try it out.

**Review by anxolin — APPROVED** [review]

**Pull request merged**